test/unit/slurm/test_slurm_config_generator/test_generate_slurm_config_files/expected_outputs/pcluster/slurm_parallelcluster_gpu_partition.conf

```diff
@@ -1,8 +1,8 @@
 # This file is automatically generated by pcluster
 
-NodeName=gpu-st-g38xlarge-[1-1] CPUs=16 State=CLOUD Feature=static,g3.8xlarge,gpu_g3.8xlarge,gpu Gres=gpu:m60:2
-NodeName=gpu-dy-g38xlarge-[1-4] CPUs=16 State=CLOUD Feature=dynamic,g3.8xlarge,gpu_g3.8xlarge,gpu Gres=gpu:m60:2
-NodeName=gpu-dy-p3dn24xlarge-[1-10] CPUs=48 State=CLOUD Feature=dynamic,p3dn.24xlarge,gpu_p3dn.24xlarge,gpu Gres=gpu:v100:8
+NodeName=gpu-st-gpug38xlarge-[1-1] CPUs=16 State=CLOUD Feature=static,g3.8xlarge,gpu_g3.8xlarge,gpu Gres=gpu:m60:2
```

Comment:
I'd change the unit tests to avoid having the instnace name int he compute name resource to be more clear that the instance type is no longer used for host names

Reply:
Done for some